examples/cosmos-sdk/transfer.md

```diff
@@ -0,0 +1,243 @@
+# Atomkraft test project
+
+## Token transfer
+
+### Prerequisite
+
+- [`python`](https://www.python.org/downloads) 3.10: For Atomkraft
+- [`pip`](https://pip.pypa.io/en/stable/installation): To install Atomkraft executable
+  - Normally after installing Python, you may just want to execute `python -m ensurepip --upgrade` to have `pip` installed.
+- [`poetry`](https://python-poetry.org/docs/#installation): For Atomkraft project
+- [`git`](https://github.com/git-guides/install-git): To clone Atomkraft template and Cosmos-SDK chain repo
+- [`make`](https://www.gnu.org/software/make): Compile Cosmos-SDK chain binary
+- [`go`](https://go.dev/doc/install): Compile Cosmos-SDK chain binary
+- [`java`](https://www.java.com/download): For [Apalache](https://apalache.informal.systems) model checker
+
+#### `ubuntu`
+
+```
+apt update -y && apt upgrade -y
+apt install pip3 git golang curl default-jre -y
+pip install --upgrade poetry
+```
+
+#### `archlinux`
+
+```
+pacman -Syu python-pip python-poetry git make go jre-openjdk gcc --noconfirm --needed
+```
+
+### Install Atomkraft
+
+```
+pip install --upgrade atomkraft
+```
+
+### Initialize project
+
+```
+atomkraft init transfer
```

Comment:
What does this line do? Just a sentence will suffice to give the reader an idea of what will happen.